sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/LargeBlobTest.groovy

```diff
@@ -70,6 +77,84 @@ class LargeBlobTest extends APISpec {
         blockList.committedBlocks.get(0).getSizeLong() == maxBlockSize
     }
 
+    @Requires({ liveMode() })
+    @Ignore("Takes really long time")
+    // This test sends payload over the wire
+    def "Upload Real Large Blob in Single Upload"() {
+        given:
+        // TODO (kasobol-msft) Bump this to 5000MB.
```

Comment:
Is this TODO meant to be part of this PR?

Reply:
And do we need to modify the tests for datalake too? Could we add a test that somehow makes sure the default max size is adhered to?

Reply:
It seems to be test account/server side issue. I'm following up on that, but I'd rather not block on this now.

Reply:
I'll take a look on datalake tests.

Reply:
@gapra-msft  I added test to `LargeFileTest.groovy` where default single upload for data lake is triggerd.

BTW. thank you for this suggestion. This prompted me to run full blown live tests on datalake file client and caught one nasty bug. I added fix for that in `DataLakeFileAsyncClient`. The confusing `remaining` method from `BufferAggregator` is also hidden and renamed to avoid this in the future.